elfi/model/elfi_model.py

```diff
@@ -208,10 +208,39 @@ def get_reference(self, name):
     def observed(self):
         return self.computation_context.observed
 
+    @property
+    def parameters(self):
+        return sorted([n for n in self.nodes if '_parameter' in self.get_state(n)])
+
+    @parameters.setter
+    def parameters(self, parameters):
+        """Set model parameters
+        
+        Note that the order will not be preserved.
```

Comment:
This was actually left there in accident. The argument was first meant to be a list.